node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const text = process.argv[2];
+
+const readText = fs.readFileSync(text, 'utf8', (err, data) => {
+  if (err) throw err;
+  return data;
+});
+
+console.log(readText);
```

Comment:
You shouldn't use the `sync` versions of the method, only `readFile`. You access the data within the callback function, so instead of `return`ing the `data` that is where you console log it.